ios/Classes/Blemulator/Model/DeviceContainer.h

```diff
@@ -0,0 +1,5 @@
+@interface DeviceContainer : NSObject
```

Comment:
PeripheralContainer?

Reply:
Wanted to keep the convention in line with Android version, hence DeviceConatiner.

Reply:
I think it's more iOS-y, but it doesn't really matter and I don't know much about the platform.